### PR TITLE
Daemons configuration (#517):

### PR DESCRIPTION
**Pull request by mohierf**
- update default configuration files
- catch daemon PortNotFree exception
- add a test for daemon starting

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ after_success:
   # to get coverage data with relative paths and not absolute we have to
   # execute coveralls from the base directory of the project,
   # so we need to move the .coverage file here :
-  - if [[ $TEST_SUITE == 'unit' ]]; then mv test/.coverage . && coveralls --rcfile=test/.coveragerc; fi
+  - if [[ $TEST_SUITE == 'unit' ]]; then mv test/.coverage . && coveralls debug && coveralls -v --rcfile=test/.coveragerc; fi
```
Comment (ddurieux) [inline .travis.yml]: We can't remove debug ?

File: alignak/http/daemon.py [hunk 2]
```diff
@@ -62,17 +62,37 @@ class HTTPDaemon(object):
...
     def __init__(self, host, port, http_interface, use_ssl, ca_cert,
                  ssl_key, ssl_cert, daemon_thread_pool_size):
-        self.port = port
-        self.host = host
-        self.srv = None
+        """
+        Initialize HTTP daemon
+
+        :param host: host address
+        :param port: listening port
```
Comment (ddurieux) [inline alignak/http/daemon.py]: add:
```:type port: int```

File: .travis.yml (hunk 1 above)
Comment (mohierf) [inline .travis.yml]: Yes we can ... but it remains an interesting information, no ? This is why I left for the moment ...

File: alignak/http/daemon.py (hunk 2 above)
Comment (mohierf) [inline alignak/http/daemon.py]: I added in my next PR ;) It may wait, no ?

**Pull request merged**